src/plugins/hathor_pubsub.js

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
You should add the config for this plugin on `src/plugins/child.js` to make it easier to enable.

Reply:
Awesome. Just added the fix.

Reply:
Done